Remove reference to windows 7 in self-signed build docs (#17192)
nvaccess/nvda: projectDocs/dev/selfSignedBuild.md

148CB69869B802A36B3D8D801BA8D9D0F3C1484F  CN=Test NVDA Build, O=NV Access Dev, C=US
```

#### Note for older versions of Windows

On any supported version of Windows, you can manage certifications through the "Certificate Manager".

Importing A Windows 11 certificate via PowerShell may fail on older versions of Windows.

> For Windows 10, simply open the certificate file and go through the installation dialog.

Reviewer: ```suggestion
On Windows 10, open the certificate file and go through the installation dialog.
```
